src/schema/schema.graphql

```diff
@@ -415,7 +415,7 @@ enum InviteType {
 }
 
 type InviteInfo {
-	type: InviteType!
+	type: InviteType #though type is a required value on the response, outdated invites come back as empty InviteInfo objects
```

Comment:
So the docs say this is required but it can actually come back as `null`? That may be a bug in the docs.

Reply:
yeah, SOAP will return `inv: [{}]` for outdated events